<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Molpher GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionNewJob"/>
    <addaction name="actionOpenSnapshots"/>
    <addaction name="separator"/>
    <addaction name="actionClearSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionLegend"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionEditJobQueue"/>
    <addaction name="actionBookmarks"/>
   </widget>
   <addaction name="menuCommands"/>
   <addaction name="menuShow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewJob"/>
   <addaction name="actionEditJobQueue"/>
   <addaction name="actionBookmarks"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEditJobQueue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Job queue</string>
   </property>
  </action>
  <action name="actionNewJob">
   <property name="text">
    <string>Create job</string>
   </property>
  </action>
  <action name="actionBookmarks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bookmarks</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to backend</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from backend</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenSnapshots">
   <property name="text">
    <string>Open snapshots</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="text">
    <string>Legend</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionClearSettings">
   <property name="text">
    <string>Reset to default settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
